### DISCUSSION TITLE
MeshCentral Monthly Meeting

### DISCUSSION THREAD

**Discussion by Ndaboom** [Ylianst/MeshCentral#6490] [General]
We thank everyone who joined yesterday's MeshCentral Community Meeting! It was great to see you all there.
We've updated our [Community Monthly Meetings page](https://github.com/Ylianst/MeshCentral/wiki/Community-Monthly-Meetings) with details on the next 12 meetings, scheduled for the 4th Thursday of each month at 2:00 PM UTC.
Missed the meeting? No worries—the recording is available in our [MeshCentral Meeting Recordings playlist](https://videos.evoludata.com/w/p/tUnLpw6z1LCASuATa7wnCo), where you can also catch up on the previous discussion.
Looking forward to seeing you all at the next one on Thursday, November 28, 2024, at 2:00 PM UTC!

**Comment by NewUserfromDE** [comment]
@Ndaboom Sammy, lessons learned from today, because 2 members had problems:

1. missing video
* Someone should ask in the chat, if everyone sees and hears the moderator.
* The meetings page could include a hint or screenshot, that the meetings are with audio and video, so that members notice missing of these.

2. firewall
* Some companies block Jitsi and comparable solutions (accidentally) at the firewall, so that video is missing. Company admins are often only used to proprietary solutions like MS Teams. Users don't notice the missing video from the start, because there is no error message, just a black video window. --> the meetings page could include a hint about firewall
(btw: this will be the same with BigBlueButton)

**Comment by NewUserfromDE** [comment]
@si458 @marclaporte

Jitsi offers an option to a) leave or b) to leave and close the session for all. Usually you close the session for all, but then the chat also immediately (!) closes and disappears, so that members have to wait for the publication of the recording to lookup hints from the chat.

(It also complicates after-meetings for interested members or to network, because they would first need to rejoin the room and they would need to know that they can do so.)

So I suggest to let the meeting open, either c) for 5 minutes after end or d) (also) write in the chat that the meeting is finished and then let it open until the last member leaves (this will end the session automatically).

@Ndaboom 

There should be an official chat message, when the meeting is finished, that the recording has stopped now. Of course it is stopped when you leave, but maybe not everyone knows this and if people decide to stay after the official recorded meeting such a message is even more important =)

Thanks again to all for the meetings. :o)

**Reply by Ndaboom** [reply]
Yeah, we should have that message, seems okay to me...Thank you.

**Reply by si458** [reply]
i was the moderator and i was expecting everyone to have left before me and i was the last person out,
so this was actually my bad! as i clicked the `end for all` when the was a few people left, i didnt realise people was still doing stuff like reading the chat!
do forgive me 🙏

**Reply by NewUserfromDE** [reply]
Absolutely no complaint! Just trying to make the meetings/community better step by step. Thanks to all =)

**Comment by NewUserfromDE** [comment]
@DaanSelen At community meetings I was involved we introduced a (regularly) part for member presentations about using or extending or modifying the software.

I guess if you would have a lightning talk (3 min) regarding meshbook no one would complain, if you present it via screen sharing at part three / last part of the monthly community meeting. Because I don't know if others read your chat hints about meshbook.

I suppose it would be no problem for you, your spoken English is dimensions better than mine. Also consider showing screenshots, although I read about meshbook it's still a bit abstract to me.

**Reply by DaanSelen** [reply]
If you can formulate some questions for me, I can answer them and create a short presentation on how it works!

**Reply by NewUserfromDE** [reply]
Question list:
(Focus on Windows, for presentation, because this is the lowest common denominator in the community.)

1. What problem does it solve? (I think it extends remote management)

"Sell" it to a windows admin, who manages 5 clients. See point 3 for explanation, because the target group is very heterogeneous. I think you don't need to explain it to someone who uses e.g. Puppet.

2. Show an example (with screenshot).

Change something, one usually has no tool for. We all know how to change registry remotely for example, regedit already has remote connection built-in, although you are right concerning firewalls. But how about automatically changing a config file or changing a user, this would be an interesting improvement of MeshCentral.

3. What are the requirements?

I see you come from Ansible and the like, but many admins still use  batch / bash scripts instead. Because often customers have small environments. Techniques like DevOps and to a degree containers still are rather techniques of large organizations and maybe will stay so. But MeshCentral is for environments of 1 client to 5000 clients, so you have administrators of all types.

---

If you'd like further discussion maybe let's open a new thread, so that this one focuses on the handling of the monthly meeting in general.

**Reply by DaanSelen** [reply]
Yeah sure open another thread. I'll answer the questions once I am home again.

**Comment by NewUserfromDE** [comment]
@Ndaboom @si458 @marclaporte 

The last MeshCentral Monthly Meeting worked well, thanks everybody. Regarding the platform I have no preference, both are open source and well established (JitsiMeet and BigBlueButton), so please keep BigBlueButton if it is easier for you and/or if you use it at work anyway.

But I have one (important) point for critique, I guess you are already aware of it:

### Preface

JitsiMeet does show the conference members on the main screen, BigBlueButton does show them at the side as a scrollable list. So it was more difficult in BigBlueButton to see who and how many members are there.

### Critique

The recording of the March meeting / BigBlueButton meeting lacks completely the side bar with the members list and the chat. The whole recording makes the impression of a very small Monthly Meeting with only 3 or 4 members. In addition all the chat information is missing like context comments to the voice chat, messages from text-only members and meta discussions. From a marketing point of view the missing side bar has IMHO a big impact, especially if users compare the first 5 meetings/recordings to the recent one, they will think the meetings are getting smaller. Plus they miss the whole chat information.

### Personal Failed Idea

To my personal regret I tried to mark the recent topic of the meeting via chat, so that it is easier for members to follow and especially for a better viewing of the recordings, since without marks it's as always a single big 1 hour block of video and if you search for a special topic then, well you search, and search - and search.

### Outlook

I personally think that the recordings are important, but also meetings without recordings, because many members of course don't want to get recorded. The chat possibility mitigates this in some way, so I always thought of the meeting as a multi media meeting: video, voice and text - and picture and slides in the future. Now I'm unsure if you see it as multi media as well or solely as a single media / video conference.

### Outlook 2

Originally I would have proposed non-recording meetings in addition to the recorded meetings in my 4 chapter strategic paper for the meetings. Since the feedback declined and also my time I stopped the writing at the moment. But as explained: to me community building is a matter of months and years, I think long-term. So this is just temporarily paused, I will attend every meeting and wait for a better suited moment. But this recording thing, no chat, no participants list, is an obstacle for me I can't ignore, because it has an steady impact over time, getting bigger each passing month.

### Thanks

Sammy, please, this is of course no personal critique for your work. Thanks for all the work as always! I would also bet, that it was not intentional. I just wanted to share my thoughts at an early point to still be able to influence.

**Reply by si458** [reply]
@Ndaboom tried to export the video with the chat but struggled so its something we need to look into more

**Reply by NewUserfromDE** [reply]
:+1: :smile: :sob:

**Reply by Ndaboom** [reply]
Yes, we can't have those recordings exported with no chats, no participants list at all, i wasn’t satisfied either with the previous meeting recording video. And i had a very busy weekend trying to figure out what tools to use to export all the video recording’s  elements(chats, videos, list of participants, etc), also tried to built one but didnt really work. But, we'll find a solution to that very soon. Thank you very much for pointing this out.

**Reply by NewUserfromDE** [reply]
I'm sorry Sammy that this is so difficult. Thanks again for your effort. Maybe the April meeting recording will be one more without chat and participants list. I'm glad that you all agree that these are important, thanks. =)

**Reply by JohnXLivingston** [reply]
Hi everyone.

There is a plugin to export more easily the recording as a video. We plan to install it very soon, so we can test. I hope it will help.

**Reply by JohnXLivingston** [reply]
It is setup!

Now, when you record an meeting on our BigBlueButton instance, you will have several formats:

* presentation: this is usefull to see a replay in a web browser (you have all video source, and the chat, bug this only works in browser). This was already available, but i'm not sure that we will make those links public.
* notes: Makes the shared notes from the meeting available as a document.
* screenshare: Generate a single video file from the screensharing and meeting audio.
* podcast: Generate an audio-only recording.
* video: Generate a recording containing the webcams, presentation area, and screensharing combined into a single video file.

![image](https://github.com/user-attachments/assets/8954b755-584f-49d1-8d11-4dc78598eec8)

I hope this will help.

**Reply by NewUserfromDE** [reply]
Wow, that was fast, merci. Looking forward to the April meeting.

**Reply by JohnXLivingston** [reply]
> Wow, that was fast, merci. Looking forward to the April meeting.

It was already on our TODO-list ;)

**Reply by JohnXLivingston** [reply]
Concerning the display, there are several modes.

(sorry, screenshots are in french...)

First, open the last menu item in the upper right menu:
![image](https://github.com/user-attachments/assets/5d08a124-1f5d-40ba-ab4e-fa99d75083a0)

Then you can choose the display mode:
![image](https://github.com/user-attachments/assets/c0141d9a-132e-41d8-8170-06a555db8a2f)

The presentator can apply this change for everyone in the room.


I don't know if this affects the recording.

**Comment by Ndaboom** [comment]
🛎️ MeshCentral Community Monthly Meeting 🛎️

Hello everyone,
This is a reminder that our next community meeting is coming up next Thursday, April 24th, in just five days. Prepare for this great event, where we will discuss project updates, potential upcoming features, community contributions, and get feedback from everyone. We will also review stalled PRs and cover any other topics related to the MeshCentral project you’d like to bring up!
We look forward to seeing you all there: Thursday, April 24, 2025, at 14:00 UTC (2 PM UTC).
To add this event and upcoming ones to your calendar, please download this ICS file at https://github.com/Ndaboom/MeshCentral-Monthly-Community-Meeting/blob/60fdae8e1ccf7107c4ed51603d45ab98b3a5a82e/meshcentral_meetings.icss, then import it to your calendar app.
For further details about the meeting, please: https://github.com/Ylianst/MeshCentral/wiki/Community-Monthly-Meetings

**Reply by NewUserfromDE** [reply]
@Ndaboom @si458 @marclaporte @DaanSelen

FYI, it seems that I can't attend the meeting tomorrow (24.7.) at all or only for some parts of it. So if I don't answer, don't be puzzled, I think I will attend as a "ghost", but I may be afk for quite some times. I would like to have helped again in structuring the meeting, I'm sorry.

**Reply by si458** [reply]
@NewUserfromDE perfectly fine! If u have anything you would like us to bring up or talk about etc then giz a quick msg with them!
